tests/common/plugins/psu_controller/snmp_psu_controllers.py

```diff
@@ -9,40 +9,90 @@
 from controller_base import run_local_cmd
 
 
-def get_psu_controller_type(psu_controller_host):
+try:
+    from pysnmp.proto import rfc1902
+    from pysnmp.entity.rfc3413.oneliner import cmdgen
+    from pyasn1.type import univ
+    has_pysnmp = True
+except:
+    has_pysnmp = False
```

Comment:
i do not feel this try expect block is neccessary, we should have pysnmp installed in the sonic mgmt docker. the tests should not worry about this.

Reply:
fixed this.